client/config/cmd_test.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+)
+
+var ErrWrongNumberOfArgs = fmt.Errorf("wrong number of arguments")
+
+func Test_runConfigCmdTwiceWithShorterNodeValue(t *testing.T) {
```

Comment:
This is boilerplate for cmd testing. You can find cmd testing [here](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/8179-add-env-vars/server/util_test.go) as well. What kind of automatic testing do we want to  implment here?

Reply:
Good idea to add os.setEnv tests for clietn config too